src/apis/LanguageLayer.js

```diff
@@ -0,0 +1,23 @@
+const { APIWrapper } = require('../')
+const fetch = require('node-fetch')
+
+const API_URL = 'http://api.languagelayer.com/'
+
+module.exports = class TumblrAPI extends APIWrapper {
```

Comment:
Tubmlr?

Reply:
shit, I forgot to remove that, sorry

Reply:
I have already fixed it, tell me if there is something else 😓